app/helpers/Intersect.js

```diff
@@ -253,19 +253,19 @@ function binarySearchForTime(array, targetTime) {
  *        from the authority (e.g. the news url) since we get that in the same call.
  *        Ideally those should probably be broken up better, but for now leaving it alone.
  */
-export function checkIntersect() {
+export async function checkIntersect() {
   console.log(
     'Intersect tick entering on',
     isPlatformiOS() ? 'iOS' : 'Android',
   );
 
-  asyncCheckIntersect().then(result => {
-    if (result === null) {
-      console.log('[intersect] skipped');
-    } else {
-      console.log('[intersect] completed: ', result);
-    }
-  });
+  const result = await asyncCheckIntersect();
+
+  if (result === null) {
```

Comment:
While we're here - could we just simplify this to

```js
console.log(`[intersect] ${result ? 'completed' : 'skipped'}`)
```

Think it would be nice to see it's just a console.log immediately rather than trying to read through the if/else.